host-configs/rzvernal-toss_4_x86_64_ib_cray-clang@14.0.0_hip.cmake

```diff
@@ -60,13 +60,15 @@ set(ENABLE_MPI ON CACHE BOOL "")
 
 set(ENABLE_HIP ON CACHE BOOL "")
 
-set(HIP_ROOT_DIR "/opt/rocm-5.1.1/hip" CACHE STRING "")
+set(HIP_ROOT_DIR "/opt/rocm-5.2.3/hip" CACHE STRING "")
 
-set(HIP_CLANG_PATH "/opt/rocm-5.1.1/hip/../llvm/bin" CACHE STRING "")
+set(HIP_CLANG_INCLUDE_PATH "/opt/rocm-5.2.3/hip/../llvm/lib/clang/14.0.0/include" CACHE PATH "")
```

Comment:
I swapped `HIP_CLANG_PATH` for `HIP_CLANG_INCLUDE_PATH` based on our `raja` recipe. 

@white238 -- Does this change seem reasonable to you? I was getting an error message the other way, but I'm not sure if it was related to this change.